### PR TITLE
fix: pass chain call options to chat_docs

### LINKED ISSUE #721: need to pass chain call options to chat_docs

### PR DESCRIPTION
**Pull request by bjwswang**
#### Which issue(s) this PR fixes

Fixes #721 

#### Special notes for your reviewer

This is only a temporary fix. We should consider to provide a consistent way to chat wiht documents just like other common chat in appruntime. Let's do this in v0.3

### CODE REVIEW

File: apiserver/pkg/chat/chat_docs.go [hunk 1]
```diff
@@ -210,13 +211,13 @@ func (cs *ChatServer) SummarizeConversationDoc(ctx context.Context, req Conversa
 		klog.V(5).Infof("Generate embeddings from file %s to vectorstore for conversation %s", doc.Filename, req.ConversationID)
 		errEmbedding = cs.GenerateSingleDocEmbeddings(ctx, req, doc, documents)
 		if errEmbedding != nil {
+			klog.V(1).ErrorS(errEmbedding, "ErrEmbedding", "document", doc.Filename, "conversation", "")
```
Comment (Abirdcfly) [inline apiserver/pkg/chat/chat_docs.go:214]:
```suggestion
			klog.V(1).ErrorS(errEmbedding, "ErrEmbedding", "document", doc.Filename, "conversation", req.ConversationID)
```
maybe you want this?

File: apiserver/pkg/chat/chat_docs.go [hunk 2]
```diff
@@ -210,13 +211,13 @@ func (cs *ChatServer) SummarizeConversationDoc(ctx context.Context, req Conversa
...
 			errStr += fmt.Sprintf(" ErrEmbedding: %s", errEmbedding.Error())
 			// break once error occurs
 			ctx.Done()
 			return
 		}
-		klog.V(1).ErrorS(errEmbedding, "ErrEmbedding", "document", doc.Filename, "conversation", "")
-		klog.V(5).Infof("Generate embeddings for doc %s is successful!")
+		klog.V(5).Infof("Generate embeddings for doc %s is successful!", doc.Filename)
```
Comment (Abirdcfly) [inline apiserver/pkg/chat/chat_docs.go:220]: IMO, Should add req.ConversationID to the logs to make it easier to locate each request run for debugging purposes

**Review by Abirdcfly — APPROVED** [review]

File: apiserver/pkg/chat/chat_docs.go (hunk 2 above)
Comment (bjwswang) [reply apiserver/pkg/chat/chat_docs.go]: Done

Comment (bjwswang) [reply apiserver/pkg/chat/chat_docs.go]: Done

**Pull request merged**